components/popup/src/vwc-popup-base.ts

```diff
@@ -98,23 +121,36 @@ export class VWCPopupBase extends LitElement {
 
 	override connectedCallback(): void {
 		super.connectedCallback();
-		document.addEventListener('scroll', this.updatePosition);
+		// Save the initial display state so that it can be restored when the positioning is complete
+		this.initialDisplayState = this.open;
+		this.hide();
+		this.anchorEl = this.getAnchorById();
+		this.anchorEl?.addEventListener('change', this.updatePosition);
```

Comment:
when is this triggered?
I think change only apply to `<input>`, `<select>`, and `<textarea>`